### ISSUE TITLE
module 'math' has no attribute 'comb'

### ISSUE THREAD

**Issue by littlebird-0904** [3b1b/manim#1808]
### Code and Error
**Code**:

**Error**:

### Environment
**OS System**: windows10
**manim version**: master 
**python version**:3.7
when I try to run the example_scenes.py, there report an error which is:
" File "C:\software\python\python3.7\lib\site-packages\manimlib\utils\simple_functions.py", line 13, in choose
    return math.comb(n, k)
AttributeError: module 'math' has no attribute 'comb'".
It seems i need to use python3.8, but i don't want to change the python version. So is there has another way to fix it?

**Comment by sennhvi** [comment]
The comb function is new in Python 3.8 (see documentation [here](https://docs.python.org/3/library/math.html)). Your Python installation is 3.7.1. If you update your Python installation, the issue should be resolved.

from [stackoverflow](https://stackoverflow.com/questions/67121494/python-attributeerror-module-math-has-no-attribute-comb)

**Comment by ishandutta2007** [comment]
I think we should update [docs ](https://github.com/3b1b/manim#installation)from `3.7.0 or higher` to `3.8.0 or higher`

**Comment by TonyCrane** [comment]
> I think we should update [docs ](https://github.com/3b1b/manim#installation)from `3.7.0 or higher` to `3.8.0 or higher`

Yes, but I think choose function should be implemented in other way. It's unnecessary to raising the lowest python version to 3.8 due to this small function.

**Comment by OuHangKresnik** [comment]
So ? How to resolve this issue ? Upgrade the version ?

**Comment by littlebird-0904** [comment]
I use the same function in the scipy. It works.



---Original---
From: ***@***.***&gt;
Date: Fri, Sep 2, 2022 00:25 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [3b1b/manim] module 'math' has no attribute 'comb' (Issue #1808)




 
So ? How to resolve this issue ? Upgrade the version ?
